lib/console_yasttest.pm

```diff
@@ -17,9 +17,9 @@ sub post_fail_hook {
 
     show_tasks_in_blocked_state if ($defer_blocked_task_info);
 
+    $self->save_upload_y2logs;
```

Comment:
This change seems undermotivated and unrelated to your commit. Please make that an explicit commit or PR or revert. Also I doubt this is a good idea as we do the 'remount_tmp_if_ro' to actually be able to save these logs when the root fs turns read-only.

@rwx788 am I right/wrong?

Reply:
Missed that, it's good point. So actually we need to move `upload_logs` call so that we try to recover network before trying to upload anything.

Reply:
Ok, will do that.
```
    $self->save_upload_y2logs;
    $self->remount_tmp_if_ro;
    upload_logs('/var/log/zypper.log');
    $self->save_system_logs;
    $self->save_strace_gdb_output('yast');
```
Then I will remove that change form here and we can put it in a different commit or PR.

Reply:
We need it like 
```perl
    $self->remount_tmp_if_ro; 
    $self->save_upload_y2logs;
    upload_logs('/var/log/zypper.log');
```

As we use /tmp to save logs.

Reply:
Done